ktor-client/ktor-client-features/ktor-client-tracing-stetho/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
and this feature can be a child of `ktor-client-tracing`, like for serialization

Reply:
Done.